tests/test_exec.py

```diff
@@ -103,99 +103,12 @@ def test_exec_additional_gids():
         shutil.rmtree(tempdir)
     return 0
 
-NONKM_OK_CONF = "var/lib/krun/config"
```

Comment:
You should be able to remove these statements near the beginning of this file:
+import hashlib
+import datetime

github won't let me put the comment where it would be actually helpful.